viewer/packages/api/src/api-helpers/Image360ApiHelper.ts

```diff
@@ -245,6 +255,7 @@ export class Image360ApiHelper {
       target: new THREE.Vector3(0, 0, -1).applyQuaternion(rotation).add(position)
     });
     this._domElement.removeEventListener('keydown', this._domEventHandlers.exit360ImageOnEscapeKey);
+    this._imageCollection!.events.image360Exited.fire();
```

Comment:
This will not work sadly :( 

This use case would brake:
1. User adds a collection A
2. User adds a collection B
3. User clicks an image from Collection A -> event gets fired for collection B :(